src/main/router/window-router.ts

```diff
@@ -0,0 +1,26 @@
+import { Router } from "../lib/route-pass/Router";
+import { mainWindow } from "../main";
+
+Router.respond("window::maximize", (_evt) => {
+  if (mainWindow.isMaximized()) {
+    mainWindow.unmaximize();
+    return;
+  }
+  mainWindow.maximize();
+});
+
+Router.respond("window::maximized", (_evt) => {
```

Comment:
Just to be consistent with the Electron API and avoid the confusion with `window::maximize`
```suggestion
Router.respond("window::isMaximized", (_evt) => {
```